### PR TITLE
Code examples for episode 043 & 044

### PR DESCRIPTION
**Pull request by int3l**

### CODE REVIEW

File: ep043/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# [introduction to tox (beginner - intermediate)](https://youtu.be/75WBE_qbpGk)
+
+Today I explain the basics of using `tox` as an automation / testing tool!
+
+## Setup commands
+
+```bash
+git clone git@github.com:asottile/pyupgrade.git
```
Comment (asottile) [inline ep043/README.md:8]: I don't use `.git` on these -- it's not necessary

Comment (int3l) [reply ep043/README.md:8]: Do you want me to remove it? I just copied what GitHub gives me by default from the `Add Code` button.

Comment (asottile) [reply ep043/README.md:8]: yes please!

**Review by asottile — APPROVED** [review]

**Pull request merged**